pcsx2/DEV9/Win32/tap-win32.cpp

```diff
@@ -400,14 +400,13 @@ bool TAPGetWin32Adapter(const std::string& name, PIP_ADAPTER_ADDRESSES adapter,
 	//Note that we append to the collection during iteration
 	for (size_t vi = 0; vi < searchList.size(); vi++)
 	{
-		int targetIndex = searchList[vi];
-
 		for (ULONG i = 0; i < table->NumEntries; i++)
 		{
+			const int targetIndex = searchList[vi];
```

Comment:
constant value, shouldn't really be in the loop (searchList doesn't change in this loop) else you're constantly creating and destroying it.